### ISSUE TITLE
Replace textures with free ones

### ISSUE THREAD

**Issue by Calinou** [RobertZenz/minetest-australopithecus-artisanry#1]
The current textures are non-free, either the Summerfields textures must be relicensed, or either we must make new ones.

http://freedomdefined.org/Licenses/NC

**Comment by RobertZenz** [comment]
Well, they are free, but not as free as one would wish I guess.

Now that you say it, CC-BY-NC-SA 4 is *guess* of mine, I opened [#156](https://github.com/SummerFields/SummerFields/issues/156) because I already had a hard time finding this note.

**Comment by Calinou** [comment]
> Well, they are free

They are not free culture, which is what I meant. Free culture is a work which you can use for any purpose, modify and distribute it for both commercial and non-commercial purposes.

**Comment by RobertZenz** [comment]
Absolutely true. But first I'd like to get a confirmation that that is the correct license.

**Comment by lipki** [comment]
I confirm that Summerfields is licensed under CC BY-NC-SA.
And yes it is not free, sorry.

I am not the original author of the pack.
So I can not take the desicion to make it truly free.

I will try to contact the author, maybe it allow me to do so.
Give me a few days :)

**Comment by RobertZenz** [comment]
@lipki I guess you haven't heard anything by now?

**Comment by RobertZenz** [comment]
With c72bfb7 I replaced the default textures with one from the Kenny Donation pack which are public domain. I also moved the Summerfields textures to their own directory so that they can be easily reinstantiated if needed.

@lipki I'd like to take the opportunity and thank you for picking up Summerfields and continuing to work on it. Thank you! Summerfields is an awesome texture pack and I'll continue to use it. Calinou is right that the NC part doesn't go well with the otherwise BSD licensed code, so the default are now the ones from Kenny with Summerfields being optional. However, I'll happily keep using Summerfields for everything and offer them optionally at least, it's just too awesome!

Also I checked that the textures can be overridden by other mods, and that's the case, so a subgame using Artisanry can easily put their own.
